sources/lib/api/gcp/bigquery/_udf.py

```diff
@@ -13,34 +13,27 @@
 """Google Cloud Platform library - BigQuery UDF Functionality."""
 
 import json
+import gcp.storage
 
 
 class FunctionCall(object):
```

Comment:
I'll remove it, but once again though I feel I should point out that in-browser testing is what the BQ team themselves promote. So if it is a cheap hack its a cheap hack that the company promotes.

http://storage.googleapis.com/bigquery-udf-test-tool/testtool.html